### PR TITLE
[7.0] Added base_multi_tag module

### PR DESCRIPTION
**Pull request by katyukha**

### CODE REVIEW

**Comment by gurneyalex** [comment]
please fix the lint issues.

File: base_multi_tag/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+    "name": "Tags",
+    "version": "0.0.3",
+    "author": "Management and Accounting On-line",
```
Comment (gurneyalex) [inline base_multi_tag/__openerp__.py]: add "Odoo Community Association (OCA)"

File: base_multi_tag/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    "website": "",
+    "summary": "Tags",
+    'category': 'Hidden/Dependency',
+    "depends": [
+        "base",
+        "base_action_rule",
+    ],
+    "description": """
+        Provides generic tags logic and basic views.
+        Integration with other modules should be implemented as sparate modules
```
Comment (gurneyalex) [inline base_multi_tag/__openerp__.py]: is there an explanation of the required steps to do so somewhere?

**Comment by gurneyalex** [comment]
@katyukha I'm sorry for missing the update you did on this. Can you rebase if you're still interested in this PR (or close it)?

**Comment by katyukha** [comment]
We use original version of [base_tags](https://github.com/katyukha/base_tags) in our 7.0 codebase, and have no plans to port it to renamed `base_multi_tag` version. So i close this PR

**Pull request closed without merging**